src/styles/index.scss

```
@@ -0,0 +1,5 @@
+@import "../styles/utils/variables";
+
+@import "../styles/page.scss";
+@import "_header.scss";
+@import "_catalog.scss";
```

My suggestion would be using same style for naming the files and path to the files as well.  
```suggestion
@import "../styles/page.scss";
@import "../styles/header.scss";
@import "../styles/catalog.scss";
```